src/test/py/bazel/foo.py

```diff
@@ -0,0 +1,18 @@
+# standard libraries
```

Comment:
If this is a test is should be name this way. It should be structured similar to other tests - arrange, act, assert. There are not asserts here.

Reply:
naming fixed.